examples/model_compress/main_filter_pruner.py

```diff
@@ -0,0 +1,173 @@
+import math
```

Comment:
please merge `main_filter_pruner.py` and `main_slim_pruner.py` into one file

Reply:
they are different paper's implementation, it will be really confusing if merged

Reply:
ok, at least change the file name, the name can be `filter_torch_vgg.py`